Block/Info.php

```diff
@@ -53,8 +55,16 @@ protected function _prepareSpecificInformation($transport = null)
     public function displayPaymentMethodTitle()
     {
         $info = $this->getInfo();
+        $token = $info->getAdditionalData();
+
         $boltProcessor = $info->getAdditionalInformation('processor');
-        if (empty($boltProcessor) || $boltProcessor == \Bolt\Boltpay\Helper\Order::TP_VANTIV) {
+        //this check must be done first as applepay is not a processor. If done with the rest of the alternative
+        //processors after the vantiv check then the display would be wrong.
+        if (!empty($token) && $token == "applepay")
+        {
+            $paymentTitle = 'Bolt-' . strtoupper($token);
```

Comment:
Perhaps we could replace `strtoupper` with `ucfirst`, here and several lines below, for consistency with `Helper\Order\TP_MEHOD_DISPLAY`

Reply:
Agreed, done